seqr/views/apis/variant_search_api.py

```diff
@@ -12,8 +12,8 @@
 from reference_data.models import GENOME_VERSION_GRCh37
 from seqr.models import Project, Family, Individual, SavedVariant, VariantSearch, VariantSearchResults, ProjectCategory, \
     Sample
-from seqr.utils.elasticsearch.utils import get_es_variants, get_single_es_variant, get_es_variant_gene_counts
-from seqr.utils.elasticsearch.constants import XPOS_SORT_KEY, PATHOGENICTY_SORT_KEY, PATHOGENICTY_HGMD_SORT_KEY
+from seqr.utils.search.utils import get_es_variants, get_single_es_variant, get_es_variant_gene_counts
+from seqr.utils.search.elasticsearch.constants import XPOS_SORT_KEY, PATHOGENICTY_SORT_KEY, PATHOGENICTY_HGMD_SORT_KEY
```

Comment:
Is it possible for the search API functions not to use the `es` functions directly? Let the search utility functions encapsulate the details of the search databases.

Reply:
see above